dataset_preprocessing/image_matcher.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np 
+import cv2
+from matplotlib import pyplot as plt
+import statistics
```

Comment:
I don't think you use matplotlib, statistics, pandas, or scipy.stats in this file.

Reply:
removed [efb3969](https://github.com/lolzballs/mie429/pull/2/commits/efb3969195e1f183a916d19df838c0aac995ff6a)